src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -92,6 +108,7 @@ public function getFunctions(): array
             new TwigFunction('impersonation_exit_path', $this->getImpersonateExitPath(...)),
             new TwigFunction('impersonation_url', $this->getImpersonateUrl(...)),
             new TwigFunction('impersonation_path', $this->getImpersonatePath(...)),
+            new TwigFunction('user_is_granted', $this->userIsGranted(...)),
```

Comment:
As we changed the name of the PHP method here #59214:

```suggestion
            new TwigFunction('is_granted_for_user', $this->userIsGranted(...)),
```

Reply:
Updated